.github/workflows/dag-push-staging.yaml

```diff
@@ -63,7 +64,14 @@ jobs:
             --machine-type    t2a-standard-32 \
             --num-nodes       1
 
-          ./scripts/setup-cluster.sh ${SERVICE_ACCOUNT}
+      - uses: imjasonh/gke-auth@v0.2.0
```

Comment:
FWIW I don't think we need this since `clusters create` also updates the kubeconfig to point to the newly created cluster. But it also doesn't hurt, so 🤷